sdk/spring/azure-spring-cloud-stream-binder-servicebus-core/src/main/java/com/azure/spring/servicebus/stream/binder/ServiceBusMessageChannelBinder.java

```diff
@@ -42,6 +44,8 @@
 
     protected static final String EXCEPTION_MESSAGE = "exception-message";
 
+    private static final ExpressionParser EXPRESSION_PARSER = new SpelExpressionParser();
```

Comment:
so, here you create a new expression parser here, is it possible to inject one?

Reply:
Maybe we don't need to inject one here, since the parser is only used in this class and only used for parsing this `"headers['" + BinderHeaders.PARTITION_HEADER + "']"` expression.

Reply:
oaky